src/index.html

```diff
@@ -11,7 +11,154 @@
+
+        <label class="form-field">
+          Surname:
+          <input type="text"
+          name="Surname"
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          Name:
+          <input type="text"
+          name="Name"
+          autocomplete="off"
+          >
+        </label>
+
+        <label class="form-field">
+          How old are You?
+          <input type="number" name="age"
+          min="1"
+          max="100"
+          value="12"
+          >
+        </label>
+
+
+        <label class="form-field">
+          Full date of birth:
+          <input type="date" name="birth-date">
+        </label>
+
+        <label class="form-field_last">
+          I accept the term of the agreement
+          <input type="checkbox" name="terms">
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Registration</legend>
+
+        <label class="form-field">
+          E-mail:
+          <input type="email"
+          name="email"
+          required
+          placeholder="email@example.com"
+          >
+        </label>
+
+        <label class="form-field_last">
+          Password:
+          <input type="password"
+          name="password"
+          required
+          minlength="8"
+          maxlength="30"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>An interesting fact about you!</legend>
+
+        <div class="form-field">
+          Do you love cats?
+
+          <label>
+            <input type="radio"
+            name="Yes"
+            value="yes"
+            >
+            Yes
+          </label>
+
+          <label>
+            <input type="radio"
+            name="No"
+            value="no"
+            >
+            No
+          </label>
+        </div>
+
+        <label class="form-field">
+          What is your favorite color?
+          <input type="color">
+        </label>
+
+        <label class="form-field">
+          What time do you go to bed?
+          <input type="time" name="bedtime">
+        </label>
+
+        <label class="form-field">
+          What are your favorite brand of cars?
+
+          <select name="cars"
+          required
+          multiple
+          >
+
+            <option value="BMW">BMW</option>
+            <option value="Audi">Audi</option>
+            <option value="Lada">Lada</option>
+          </select>
+        </label>
+
+        <label class="form-field_last">
+          How do you rate our work?
+          <input type="range" name="rating"
+          min="0"
+          max="100"
+          value="0"
+          >
+        </label>
+      </fieldset>
+
+      <fieldset class="fieldset">
+        <legend>Additional info:</legend>
+
+        <label class="form-field">
+          Comments:
+          <textarea name="comments"
+          autocomplete="off"
+         ></textarea>
```

Comment:
<img width="1327" alt="Screenshot 2023-01-18 at 18 14 03" src="https://user-images.githubusercontent.com/53228259/213230507-0eae6dd8-fd70-4576-99d3-0b04a1181517.png">

Reply:
Thanks for reviewing - I think I didn't get your question.